src/__tests__/Utils/ConfirmOverlay.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import ConfirmOverlay from "../../Utils/ConfirmOverlay/ConfirmOverlay";
+let instance;
+
+test('On submit callback test', () => {
+    const callbackMock = jest.fn();
```

Comment:
Please extract this to a `beforeEach()` method. Reference: `__tests__/ProductView/QueryInterface.js`